uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/services/document.js

```diff
@@ -242,6 +272,41 @@ define([
           });
 
           return deferred.promise;
+        },
+        CacheContactsAndAssignments: function (documentList, options) {
+          var contactIds = [];
+          var assignmentIds = [];
+
+          angular.forEach(options, function (type) {
+            switch (type) {
+              case 'contacts':
+                contactIds = collectContactIds(documentList);
+
+                if (config.CONTACT_ID) {
+                  contactIds.push(config.CONTACT_ID);
```

Comment:
Yes there was. And as we saw to call to fetch contact details of currently logged in user and caching it, is already done in `uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/modules/run.js`. there is no need to fetch the contact details and cache again. So, removed from here.
```js
    ContactService.get({'IN': [config.LOGGED_IN_CONTACT_ID]}).then(function (data) {
        ContactService.updateCache(data);
      });
```